upgrade-instructions.md

```diff
@@ -12,6 +12,27 @@ software documentation. Do not follow this guide blindly!
 
 Note: this section is in progress. It will be updated during all the development process until the release.
 
+- [Set up TTL on the mailbox_change and email_change tables](#set-up-ttl-on-the-mailboxchange-and-emailchange-tables)
+
+### Set up TTL on the label_change table
+Date: 15/09/2023
+
+JIRA: https://issues.apache.org/jira/browse/JAMES-3940
+
+Concerned products: Distributed James, Distributed James ESv6
+
+From now on, we set a default 60 days (configurable) Cassandra time-to-live on the `label_change` table
+to reduce considered outdated changes' storage usage.
+
+If one want to keep the old behavior which is not using Cassandra time-to-live on those JMAP change tables, please set the
+following configurations in `cassandra.properties`:
+
+```
+label.change.ttl=0
+```
```

Comment:
You don't precise that the value is in seconds. And should it be seconds or days btw?

Reply:
It defaults to `second`. That is good enough for OPS IMO.

Reply:
Maybe with the configuration doc? Or some examples? It's not trivial that if you put just a number it's in seconds and that you can add the unit of duration as well alongside it for example :)